syncstorage/src/db/transaction.rs

```diff
@@ -134,7 +135,8 @@ impl DbTransactionPool {
                         self.collection.clone(),
                         self.bso_opt.clone(),
                     )
-                    .await?;
+                    .await
+                    .map_err(ApiError::from)?;
```

Comment:
Looks like this'll continue working:
```suggestion
                    .await?;
```